modules/microshift-uninstalling-lvms-csi-driver.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-uninstalling-lvms-csi-driver_{context}"]
+= Uninstalling the CSI driver implementation
+
+To uninstall the installed CSI driver implementation, use the following procedure.
+
+.Prerequisites
+
+* {microshift-short} is installed and running.
+* You have deployed the CSI driver implementation.
```

Comment:
Since it's deployed when there is no user input, it may be confusing to suggest the users performed the action.

```suggestion
* The CSI driver implementation is deployed on the MicroShift cluster.
```

Reply:
done